content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -295,14 +295,14 @@ cluster.
 
```

Comment:
For the previous line, we'd prefer a lowercase `d` (we [write headings in sentence case](https://kubernetes.io/docs/contribute/style/style-guide/#headings)).